packages/terra-form-validation/package.json

```
@@ -25,12 +25,12 @@
     "final-form": "^4.6.0",
     "react": "^16.8.5",
     "react-dom": "^16.8.5",
-    "react-final-form": "^5.0.2"
+    "react-final-form": ">=5.0.2"
```

Should we cap this at v6? If v7 is released, this would accept it.

```suggestion
    "react-final-form": ">=5.0.2 <7.0.0"
```

---

Yeah, that sounds good